config/default.yml

```diff
@@ -1724,6 +1724,11 @@ Lint/Debugger:
       - debug/open
       - debug/start
 
+Lint/DefinedWithFixedResult:
```

Comment:
feels like the name of the cop should convey that the result is not only fixed, but also unexpectedly truthy.

"always defined argument"? "ineffective defined argument"? 🤷‍♂️

Reply:
I renamed it to `UselessDefined`, there are already quite a few `Useless*` cops

Reply:
I also prefer the name `UselessDefined`.